run-test.py

```diff
@@ -924,6 +1029,8 @@ def check_dependencies():
     test_stop_start_heavy(HEAVY, pod_ips)
     test_stop_start_heavy(HEAVY, pod_ips, restore_from_backup = True)
 
+    check_abandoned_requests(verbose=True)
+
```

Comment:
Shouldn't it be checked after any run of a benchmark?

Reply:
UPD: ok, please rename it to something like `check_abandoned_counter_is_not_increasing`